app/views/shared/_focus.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="someFocus">
+    <div class="someFocusText">
+      <% if local_assigns[:data_type] == 'category' && (!local_assigns[:profile] || local_assigns[:profile] =current_user.id) %>
```

Comment:
`local_assigns[:profile] =current_user.id` should be `local_assigns[:profile] == current_user.id`

Looks like this an issue in multiple places.

Reply:
Yah, just noticed.. I fixed this